site/source/tutorials/005_form_widgets/index.md

```diff
@@ -29,7 +29,33 @@ The first step to allowing the user to create new workers is to create a form. T
 
 {% instruction 'Add the following to `WorkerForm.ts`.' %}
 
-{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:1-4,7,14-15,19-27,29,42-43,48-51,85-87 %}
+```typescript
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { v } from '@dojo/widget-core/d';
+import { ThemedMixin, theme } from '@dojo/widget-core/mixins/Themed';
+import * as css from '../styles/workerForm.css';
+
+export interface WorkerFormProperties {
+}
+
+export const WorkerFormBase = ThemedMixin(WidgetBase);
+
+@theme(css)
+export default class WorkerForm extends WorkerFormBase<WorkerFormProperties> {
+
+	private _onSubmit(event: Event) {
+		event.preventDefault();
+	}
+
+	protected render() {
+		return v('form', {
+			classes: this.theme(css.workerForm),
+			onsubmit: this._onSubmit
+		}, [
```

Comment:
odd formatting, also do we need the empty children?

Reply:
I removed the children. What else is odd about the formatting?